lib/models/Order.js

```diff
@@ -23,6 +23,32 @@ class Order {
     // { id: '1', quantityOfItems: 10 }
     return new Order(rows[0]);
   }
+
+  static async selectAll() {
+    const { rows } = await pool.query(
+      'SELECT id, quantity_of_items FROM orders'
```

Comment:
to simplify you could just say `'SELECT * FROM orders'`

the * is like saying SELECT **all**

Reply:
Yes, tough to break habits from Foundations 2. Thanks for the tip!